src/main/resources/views/settings.fxml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.TextField?>
+
+<?import javafx.scene.layout.Region?>
+<AnchorPane fx:id="wrapper" prefHeight="200.0" prefWidth="300.0"
+            xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1"
+            fx:controller="controllers.SettingsController">
+    <VBox alignment="CENTER" AnchorPane.bottomAnchor="0.0"
+          AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+        <padding>
+            <Insets left="15.0" right="15.0"/>
+        </padding>
+
+        <HBox alignment="CENTER">
+            <HBox>
+                <CheckBox fx:id="chkBoxLimitFps">
+                    <HBox.margin>
+                        <Insets right="6.0"/>
+                    </HBox.margin>
+                </CheckBox>
+                <Label text="Limit FPS:"/>
```

Comment:
This label should instead be a `text` attribute on the `CheckBox`. That way it works as part of the checkbox and you don't have to exactly hit the square (and the focused graphic works better).